packages/eslint-config/package.json

```diff
@@ -27,20 +27,15 @@
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-sort-class-members": "^1.17.1"
+    "eslint-plugin-sort-class-members": "^1.17.1",
+    "prettier": "3.2.5",
+    "eslint-config-prettier": "^8.8.0 || ^9.1.0",
+    "eslint-plugin-prettier": "^5.1.3"
```

Comment:
This dependencies was in dev and peer deps that is why they was not included into yarn.lock file after installing @tinkoff/eslint-config package